hexer/Processor.cpp

```diff
@@ -86,13 +86,14 @@ void process(HexGrid *grid, PointReader reader)
 
 void processLaz(HexGrid *grid, std::ifstream& file)
 {
-    char buf[30];
-
     lazperf::reader::generic_file l(file);
 
-    const lazperf::header14 h = l.header();
     size_t count = l.pointCount();
 
+    const lazperf::header14 h = l.header();
```

Comment:
This isn't guaranteed to work in C++. It may work with your compiler, but it's not valid in general. Use a std::vector.

Reply:
OK. I'll have to change lazperf's readPoint() then (only accepts char arrays)